src/resources/postcss/tickets.pcss

```diff
@@ -507,6 +507,10 @@
 					background: var(--color-icon-error) ! important;
 				}
 			}
+
+			+ .tribe-ticket {
```

Comment:
Is this intentional `+`?

Reply:
yes it's `.tribe-ticket + .tribe-ticket` or "any `.tribe-ticket` following a `.tribe-ticket`"